server/service/src/processors/transfer/shipment/README.md

```diff
@@ -0,0 +1,19 @@
+# Shipment Transfer Processor
+
+As per general description in [transfer processors](../README.md) and these diagrams:
+
+From [TMF internal google doc](https://docs.google.com/presentation/d/1eEe0uBGvkXbYnKc2oLO2U0qRwFv4l0ws4QwFZa6e74s/edit#slide=id.p):
+
+![omSupply shipment transfer processors](./doc/omSupply_shipment_transfer_workflow.png)
+
+From [TMF internal docs](https://app.diagrams.net/#G1o_xRQAhjVsnqhxhJEu9dY6AZ_lJfG9co)
+
+![omSupply shipment transfer processors](./doc/omSupply_shipment_transfer_processors.png)
+
+## Request Requisition linked to Inbound Shipment
+
+When inbound shipment is created from outbound shipment, we check for requisition linked to outbound shipment (response requisition) and check what request requisition is linked to that response requisition and link inbound shipment with correct request requisition
```

Comment:
```suggestion
When an inbound shipment is created from outbound shipment, we check for a requisition linked to the outbound shipment (response or client requisition) and check which request requisition is linked to that response requisition and link the inbound shipment with correct request requisition
```

Reply:
Is `client requisition` a new type ? I think it's called `customer requisition` ? I was hoping to avoid using legacy terms in omSupply, added suggestion as a commit, without `or client`: 6225fa298f3bc269f623b520aed754ae1d7a6af5

Reply:
sorry, yeah. I meant customer 🙄